tasks/frontEnd/src/App.vue

```diff
@@ -0,0 +1,89 @@
+<template>
```

Comment:
What was your reason for including a lot of the api calls in `App.vue`, rather than a separate file? Or using a store based package, like Pinia?

Reply:
Only due to the small size of the test, I would want to keep App.vue as clean as possible in a real world scenario.